docs/source/bookstore_api.yaml

```diff
@@ -9,7 +9,7 @@ info:
   license:
     name: BSD 3-clause
     url: https://github.com/nteract/bookstore/blob/master/LICENSE 
-  version: 2.3.0.dev0
+  version: 2.5.1
```

Comment:
Is this the correct version bump for this?

Reply:
I was bumping it to match what the version of bookstore itself would be upon release, minus the dev0. 

We don't have a separate versioning scheme for the API (we can create one if we think that's best). I removed the dev0 simply because that when we release this, it would match the published version of the package. 

I'm open to other suggestions as to how to handle this, though.

Reply:
Might be good to have the version kept in sync between the two files programmatically when running the `bdist_wheel`. If that's a pain to do, then we should instead just add it explicitly to the RELEASING.md instructions imo.

Reply:
I think we could set up bumpversion do handle that, but I think for now the low overhead way of making it part of the RELEASING.md makes sense. I think there are a couple of other places where that would help too.

Reply:
I realize I'm coming in with a fresh take on this... The version in this YAML file could be different from the library itself since it's the REST API. 🤷‍♂